projects/saturn/src/main/resources/log4j2.properties

```diff
@@ -18,7 +18,7 @@ appender.stdout.type = Console
 appender.stdout.name = stdout
 appender.stdout.target = System.out
 appender.stdout.layout.type = PatternLayout
-appender.stdout.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} %-5p %m%n
```

Comment:
what does %t do?

Reply:
just add current thread name to the log record, needed for systems with more that one thread